### ISSUE TITLE
Maintenance: publish new version?

### ISSUE THREAD

**Issue by sudodoki** [jdan/cleaver#162]
Came into understanding version from NPM has not yet fixed issues that are available in upstream when ran into 
https://github.com/jdan/cleaver/blob/4028c1df2effdde464f16a6af58c126731e5d276/resources/default.css#L135 (z-index:1) fix for #148 not being present in resolved from npm version
```
"_resolved": "https://registry.npmjs.org/cleaver/-/cleaver-0.8.2.tgz",
"_shasum": "fc847b06ab45b0845a6a02a22eb7f112de4a0b28",
```
Default.css features
```
.progress {
  position: fixed;
  top: 0; left: 0; right: 0;
  height: 3px;
}
```
in there. Assuming publishing new version will fix it.

**Comment by sudodoki** [comment]
Also, currently in master in package.json version is 0.8.1 yet npm has 0.8.2 as well as release tab in this repo. 🙀

**Comment by sudodoki** [comment]
Fixed in 0.8.3
